tree/tree/src/TChain.cxx

```diff
@@ -1340,6 +1340,10 @@ Long64_t TChain::LoadTree(Long64_t entry)
       // (the friends of the chain will be updated in the
       // next loop).
       fTree->LoadTree(treeReadEntry);
+
+      // Propagate the IMT settings
+      fTree->SetImplicitMT(fIMTEnabled);
```

Comment:
Humm .. that's an interesting case.  In this case the `TTree` is not 'new' and thus (given the rest of this PR) the only reason that we would have `fTree->fIMTEnabled == fIMTEnabled` would be if the user did:
```
chain->SetImplicitMT(true);
chain->LoadTree(e);
chain->GetTree()->SetImplicitMT(false);
chain->LoadTree(e); (or some other entry).
```
It debatable if after this sequence we should have `chain->GetTree()->fIMTEnabled` equal to true or false.  Either way it sounds like we should document the choice.